packages/gatsby/src/commands/develop.ts

```diff
@@ -1,51 +1,74 @@
-/* @flow */
-
-const url = require(`url`)
-const glob = require(`glob`)
-const fs = require(`fs`)
-const openurl = require(`better-opn`)
-const chokidar = require(`chokidar`)
-const express = require(`express`)
-const graphqlHTTP = require(`express-graphql`)
-const graphqlPlayground = require(`graphql-playground-middleware-express`)
-  .default
-const graphiqlExplorer = require(`gatsby-graphiql-explorer`)
-const { formatError } = require(`graphql`)
-const got = require(`got`)
-const webpack = require(`webpack`)
-const webpackConfig = require(`../utils/webpack.config`)
-const bootstrap = require(`../bootstrap`)
-const { store, emitter } = require(`../redux`)
-const { syncStaticDir } = require(`../utils/get-static-dir`)
-const buildHTML = require(`./build-html`)
-const { withBasePath } = require(`../utils/path`)
-const report = require(`gatsby-cli/lib/reporter`)
-const launchEditor = require(`react-dev-utils/launchEditor`)
-const formatWebpackMessages = require(`react-dev-utils/formatWebpackMessages`)
-const chalk = require(`chalk`)
-const address = require(`address`)
-const cors = require(`cors`)
-const telemetry = require(`gatsby-telemetry`)
-const WorkerPool = require(`../utils/worker/pool`)
-
-const withResolverContext = require(`../schema/context`)
-const sourceNodes = require(`../utils/source-nodes`)
-const createSchemaCustomization = require(`../utils/create-schema-customization`)
-const websocketManager = require(`../utils/websocket-manager`)
-const getSslCert = require(`../utils/get-ssl-cert`)
-const { slash } = require(`gatsby-core-utils`)
-const { initTracer } = require(`../utils/tracer`)
-const apiRunnerNode = require(`../utils/api-runner-node`)
-const db = require(`../db`)
-const detectPortInUseAndPrompt = require(`../utils/detect-port-in-use-and-prompt`)
-const onExit = require(`signal-exit`)
-const queryUtil = require(`../query`)
-const queryWatcher = require(`../query/query-watcher`)
-const requiresWriter = require(`../bootstrap/requires-writer`)
-const {
+import url from "url"
```

Comment:
I think our imports should be `* as` style to comply with the real import. We're now relying on moduleInterop which is wrong in my opinion.

Reply:
We discussed this and think it's better to ignore the difference from commonjs and modules for a simplistic syntax.